cmd/activator/main.go

```diff
@@ -173,14 +174,14 @@ func main() {
 			pool = x509.NewCertPool()
 		}
 
-		if ok := pool.AppendCertsFromPEM(caSecret.Data["ca.crt"]); !ok {
+		if ok := pool.AppendCertsFromPEM(caSecret.Data[certificates.SecretCaCertKey]); !ok {
 			logger.Fatalw("Failed to append ca cert to the RootCAs")
 		}
 
 		tlsConf := &tls.Config{
 			RootCAs:            pool,
 			InsecureSkipVerify: false,
-			ServerName:         networkConfig.QueueProxySAN,
+			ServerName:         certificates.FakeDnsName,
```

Comment:
As I commented below (or above), it would be possible with "activator always in the path" workaround but it will be super difficult to drop the "activator always in the path" if we support it.
I think we need to consider it carefully.

Reply:
I think there are a couple options here:

1. Force "activator always in the path" for now.
2. Activator can use SNI to serve different certs on a per-connection basis (basically, activator would spoof the cert from the `queue-proxy` namespace).

For now, I'd suggest we do 1, and then add 2 in a future iteration.